extensions/amp-a4a/amp-a4a-format.md

```diff
@@ -61,6 +61,19 @@ rel="canonical">` tag.
    and won't be independently search-indexed, so self-referencing
    would be useless.
 
+1. The creative can include optional meta tags in HTML head as identifiers,
+   in the format of `<meta name="amp4ads-id" content="${namespace}:${id}">`.
+   Those meta tags have to be placed before the `amp4ads-v0.js` script.
+
+   _Rationale_: Those custom identifiers can be used to identify the impression
+   or the creative etc.. It can be helpful for reporting or debugging.
+
+   _Example_:
+   ```html
+<meta name="amp4ads-id" content="xyz-creative-id:1283474">
+<meta name="amp4ads-id" content="abc-impression-id:xIsjdf921S">
```

Comment:
I think that I'm on @lannka 's side on this.  Creative content is up to the ad networks and presumably they're the ones who will want to analyze it.  I don't think we'd want to provide anything that would allow third (fourth?) parties to do bulk analysis of ads or something.

Counter-argument: If it's standardized, then authoring / debugging tools could easily handle creatives from different networks.  E.g., I could envision a debugging panel that knows how to pull this tag out and show info to a publisher or creative author, regardless of which network originated the creative.

Not sure which concern dominates in this case, though.